omod/src/main/webapp/resources/js/custom/Custom.js

```diff
@@ -178,195 +215,240 @@ $(function(){
+            },
+            error: function(msg, status, error){
+                alert(error);
+            }
+        });
+    });
+    $("#No").click(function(){
+        updateControls(0);
+        window.location = openmrsContextPath+'/module/muzimafingerPrint/managefingerprint.form';
+    });
+    $(".doCancel").click(function(){
+        updateControls(4);
+    });
+    $("#btnUpdatePatient").click(function(){
+        var selectedPatientId = $('input[name=selectedPatient]:checked').val();
+        var jsonData = "{patient: {patientUUID : '"+selectedPatientId+"' , fingerprint :'"+ _FINGERPRINT_DATA+"'}}";
+        $.ajax({
+            url: "fingerprint/UpdatePatientWithFingerprint.form",
+            type: "POST",
+            data: jsonData,
+            contentType: 'application/json',
+            dataType: 'json',
+            async: false,
+            success: function(msg) {
+                alert("Patient updated!");
+                updateControls(0);
+            },
+            error: function(msg, status, error){
+                console.log(msg);
+                alert(error);
+            }
         });
 
-        $.validator.addMethod("checkDigit", function (value, element) {
-                    var num = value.split('-');
-                    if (num.length != 2) {
-                        return false;
-                    }
-                    return $.fn.luhnCheckDigit(num[0]) == num[1];
-                }, "Please enter digits that matches CheckDigit algorithm."
-            );
-
-        $.validator.addMethod("nonFutureDate",
-            function (value, element) { return Date.parse(value.replace("-","/")) < new Date().getTime(); },
-            "Date can not be in the future."
-        );
-
-        $.fn.luhnCheckDigit = function (number) {
-            var validChars = "0123456789ABCDEFGHIJKLMNOPQRSTUVYWXZ_";
-            number = number.toUpperCase().trim();
-            var sum = 0;
-            for (var i = 0; i < number.length; i++) {
-                var ch = number.charAt(number.length - i - 1);
-                if (validChars.indexOf(ch) < 0) {
-                    return false;
-                }
-                var digit = ch.charCodeAt(0) - 48;
-                var weight;
-                if (i % 2 == 0) {
-                    weight = (2 * digit) - parseInt(digit / 5) * 9;
-                }
-                else {
-                    weight = digit;
+    });
+
+    $("#btnCreatePatient").click(function(){
+        if($("#formData").valid()){
+            var jsonData = JSON.stringify($('#formData').serializeEncounterForm());
+            $.ajax({
+                url: "fingerprint/register.form",
+                type: "POST",
+                data: jsonData,
+                contentType: 'application/json',
+                dataType: 'json',
+                async: false,
+                success: function(msg) {
+                    alert("Patient Created!");
```

Comment:
Do we need to do the alert? Maybe create some message inline in the html instead of popup alert.